zenoh-kotlin/src/commonMain/kotlin/io/zenoh/queryable/Query.kt

```diff
@@ -48,6 +48,8 @@ class Query internal constructor(
     /** Shortcut to the [selector]'s parameters. */
     val parameters = selector.parameters
 
+    val parametersMap = selector.parametersMap
+
```

Comment:
We aim to provide users with a unified API across all the languages, for Rust, Kotlin, Python, C, C++... We'll take as reference the Rust codebase. Since this is not exposed on the Rust API (you can check it here https://docs.rs/zenoh/latest/zenoh/queryable/struct.Query.html), please remove.